src/makeInfinityAdder.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  const add = (number) => {
+    if (number !== undefined) {
+      sum += number;
+
+      return add;
+    } else {
```

Comment:
You can avoid this additional `else` condition and write code below without else {} cause you return a value above